Documentation/GraphicalEditor/Index.rst

```diff
@@ -0,0 +1,430 @@
++-----------------------------------+---------------------------------------------------------------------------------------------+
+|**Main module**                    |This is the module key of the section in the TYPO3 module menu to which the module is        |
+|                                   |assigned. For example, "web" or "site".                                                      |
++-----------------------------------+---------------------------------------------------------------------------------------------+
+|**Controller action combinations** |In each line all actions of a controller supported by this module are listed by              |
+|(Advanced options)                 |``<controllerName> => <action1>,<action2>,...``. The first action of the first line is the   |
+|                                   |default action. Actions are defined in the related aggregate root object, and the controller |
+|                                   |name corresponds to the object name.                                                         |
+|                                   |                                                                                             |
+|                                   |An example is                                                                                |
+|                                   |                                                                                             |
+|                                   |.. code-block:: none                                                                         |
+|                                   |                                                                                             |
+|                                   |   Blog => list,show,new,create,edit,update,delete,duplicate                                 |
+|                                   |   Author => list,show,new,create,edit,update,delete                                         |
+|                                   |                                                                                             |
++-----------------------------------+---------------------------------------------------------------------------------------------+
+
+.. include:: /Images/ManualScreenshots/BackendModules.rst.txt
+
+6. Save the extension
+=====================
+
+If your model represents the domain you wanted to implement you can hit the
+:guilabel:`Save` button at the top.
+The Extension Builder generates all required files for you in
+:file:`typo3conf/ext/<extension_key>/` and its installation depends on your
+local setup:
+
+6.a. Composer mode
+------------------
+
+If you run TYPO3 in :doc:`Composer mode <t3install:QuickInstall/Composer/Index>`,
+you have to specify and configure a `local path repository <https://getcomposer.org/doc/05-repositories.md#path>`_
+and copy the generated extension to this local path.
```

Comment:
move instead of copy so that composer is able to create a symlink

Reply:
Fixed in commit "[DOCS] Small fixes derived from backport of documentation".